auth_otp/views_extended.py

```
@@ -0,0 +1,310 @@
+        
+        serializer = OTPStatusSerializer(otp_request)
+        
+        return Response(
+            {
+                'success': True,
+                'data': serializer.data
+            },
+            status=status.HTTP_200_OK
+        )
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)